bluesky_kafka/__init__.py

```diff
@@ -305,3 +305,182 @@ def no_work_during_wait():
     def stop(self):
         self._consumer.close()
         self.closed = True
+
+
+class BlueskyConsumer:
+    """
+    Process Bluesky documents received over the network from a Kafka server.
+
+    There is no default configuration. A reasonable configuration for production is
+        consumer_config={
+            "auto.offset.reset": "latest"
+        }
+
+    Parameters
+    ----------
+    topics: list
+        List of topics as strings such as ["topic-1", "topic-2"]
+    bootstrap_servers : str
+        Comma-delimited list of Kafka server addresses as a string such as ``'127.0.0.1:9092'``
+    group_id: str
+        Required string identifier for Kafka Consumer group.
+    consumer_config: dict
+        Override default configuration or specify additional configuration
+        options to confluent_kafka.Consumer.
+    polling_duration: float
+        Time in seconds to wait for a message before running function work_while_waiting.
+        Default is 0.05.
+    deserializer: function, optional
+        optional function to deserialize data. Default is pickle.loads.
+
+    Example
+    -------
+
+    Print all documents generated by remote RunEngines.
+
+    >>> d = RemoteDispatcher(
```

Comment:
It looks like this example is left over from the RemoteDispatcher?

Reply:
Updated.